extensions/fp16-in-shader-and-buffer.md

```diff
@@ -0,0 +1,311 @@
+<!-- PR0 -->
+
+# FP16 usages support (PR0)
+
+**Roadmap:** This draft extension is **on the standards track**. It is currently intended to be used in Web-compliant contexts, any user with a capable device will be able to use this extension to acquire the FP16 ability. 

```

Comment:
Anything that's not described in the actual specs must not be exposed to the Web. So if we land this document, this should say something like:

```suggestion
**Roadmap:** This draft extension is **on the standards track**, but is a work in progress. User agents must not implement/expose these features until they are merged into the main specs.
```

Reply:
Done.